collectors.rb

```diff
@@ -60,17 +61,14 @@ def collect_db(db_type, host, user, password)
     when :postgres
       collect_postgres(host, user, password)
     else
-      STDERR.puts "db_type #{db_type} is neither `:mysql` nor `:postgres`."
+      STDERR.puts "db_type `#{db_type}` is neither `:mysql` nor `:postgres`."
     end
   end
 
   def collect_mysql(host, user, password)
+    # TODO Raise an error / print to stderr if something breaks?
     # Establish a connection to the database
-    begin
-      db = Sequel.connect("mysql://#{user}:#{password}@#{host}")
-    rescue
-      STDERR.puts("There was an error connecting to #{host}, u: #{user}, p: #{password}")
-    end
+    db = Sequel.connect("mysql://#{user}:#{password}@#{host}")
```

Comment:
The code was removed because it was not working. The `rescue` block was never triggered because of how `Sequel.connection(...)` works.